Requirement already satisfied: social-auth-core in /usr/local/lib/python3.8/site-packages (3.3.3)
Requirement already satisfied: six>=1.10.0 in /usr/local/lib/python3.8/site-packages (from social-auth-core) (1.15.0)
Requirement already satisfied: cryptography>=1.4 in /usr/local/lib/python3.8/site-packages (from social-auth-core) (3.1.1)
Requirement already satisfied: python3-openid>=3.0.10; python_version >= "3.0" in /usr/local/lib/python3.8/site-packages (from social-auth-core) (3.2.0)
Requirement already satisfied: requests-oauthlib>=0.6.1 in /usr/local/lib/python3.8/site-packages (from social-auth-core) (1.3.0)
Requirement already satisfied: requests>=2.9.1 in /usr/local/lib/python3.8/site-packages (from social-auth-core) (2.24.0)
Requirement already satisfied: defusedxml>=0.5.0rc1; python_version >= "3.0" in /usr/local/lib/python3.8/site-packages (from social-auth-core) (0.7.0rc1)
Requirement already satisfied: oauthlib>=1.0.3 in /usr/local/lib/python3.8/site-packages (from social-auth-core) (3.1.0)
Requirement already satisfied: PyJWT>=1.4.0 in /usr/local/lib/python3.8/site-packages (from social-auth-core) (1.7.1)
Requirement already satisfied: cffi!=1.11.3,>=1.8 in /usr/local/lib/python3.8/site-packages (from cryptography>=1.4->social-auth-core) (1.14.3)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.8/site-packages (from requests>=2.9.1->social-auth-core) (2020.6.20)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.8/site-packages (from requests>=2.9.1->social-auth-core) (1.25.10)
Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.8/site-packages (from requests>=2.9.1->social-auth-core) (2.10)
Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.8/site-packages (from requests>=2.9.1->social-auth-core) (3.0.4)
Requirement already satisfied: pycparser in /usr/local/lib/python3.8/site-packages (from cffi!=1.11.3,>=1.8->cryptography>=1.4->social-auth-core) (2.20)
